jans-cli/README.md

```diff
@@ -141,5 +144,9 @@ Now selecting 1 and it returns our desired result as below image:
 
 ![default-authentication-method.png](../docs/assets/user/using-jans-cli/images/image-im-cur-default-auth-03042021.png)
 
-So, That was a quick start to view how this _jans-cli_ Interactive Mode works. Please, follow this [link](../docs/user/using-jans-cli) to read the _jans-cli_ docs for a better understanding of the Janssen Command-Line.
+So, That was a quick start to view how this _jans-cli_ Interactive Mode works. Please, see documentation for further information.
```

Comment:
Replace with "The above was a quick start to how _jans-cli_ Interactive Mode works. Please, see documentation for further information."